consensus/src/chained_bft/epoch_manager.rs

```diff
@@ -261,10 +262,11 @@ impl<T: Payload> EpochManager<T> {
             recovery_data.root_block(),
         );
         info!("Update Network about new validators");
-        self.network_sender
-            .update_eligible_nodes(recovery_data.validator_keys())
-            .await
-            .expect("Unable to update network's eligible peers");
+        block_on(
```

Comment:
@metajack, is this safe? I remember you suggesting we do not use futures::block_on? How should we handle this? Or does this not matter since the thread / task in this case is separated from tokio by a channel?